Murcielago/Murcielago/UserInterface/Views/SettingsView/SettingsView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SettingsView.swift
+//  Murcielago
+//
+//  Created by Sam Franusic on 11/12/24.
+//
+
+import SwiftUI
+
+struct SettingsView: View {
+    @EnvironmentObject var mainModel: MainViewModel
+
+    var body: some View {
+        VStack {
+            Section(header: Text("Settings")) {
+                List {
+
+                }
+            }
+            Button(
+                action: {
+                    mainModel.signOut()
+                },
+                label: {
+                    Text("Sign Out")
+                        .frame(height: 50)
+                        .frame(maxWidth: .infinity)
+                        .overlay {
+                            RoundedRectangle(cornerRadius: 8)
+                                .stroke(lineWidth: 2.0)
+                                .padding([.horizontal], 25)
+                        }
+
+
+                }
+            )
+            versionLabel
+        }
+        .tint(.murcielagoPrimary)
+    }
+
+    private var bundleShortVersion: String {
+        Bundle
+            .main
+            .infoDictionary?["CFBundleShortVersionString"]
+        as? String ?? "x"
```

Comment:
nit: you are retrieving a string from the bundle, is it required to be formatted as SwiftUI modifiers?

Reply:
I was just line breaking it to avoid a long line, but I will look into this. Thanks for the callout!